capstone/src/main/java/com/google/sps/data/SurveyResponse.java

```diff
@@ -0,0 +1,68 @@
+package com.google.sps.data;
+
+/* This object represents each response a user submits */
+public class SurveyResponse {
+    private String id;
+    private double score;
+    private String gender;
+    private String ageRange;
+    private long responseTime;
```

Comment:
Aren't you going to have multiple response times per survey response? One survey response to 3 questions will have Response time #1 Response time #2, etc.

Reply:
Yeah, I will use the response time for the free text section only. For the MVP I plan to include the Surveys API and ensure that my classes work with the API

Reply:
You might find that the response time for the other questions are useful to analyze. I would recommend storing response times for every response or at least use a data structure here which can allow for that possibility. Otherwise you are stuck with only 1 response time per Survey Response (which involves responses to multiple questions).